src/RMGScintillatorOutputScheme.cc

```diff
@@ -15,6 +15,7 @@
 
 #include "RMGScintillatorOutputScheme.hh"
 
+#include <memory>
```

Comment:
add this to the header (actually, it is already needed there. We have `unique_ptr`s in the header, it was just forgotten)

also for germanium